modules/setting/database.go

```diff
@@ -39,6 +39,7 @@ var (
 		LogSQL            bool
 		Charset           string
 		Timeout           int // seconds
+		SQliteJournalMode string
 		UseSQLite3        bool
```

Comment:
`SQliteJournalMode`, the `SQlite` seems not the same as the word in `UseSQLite3` below.

Maybe `SQLiteJournalMode`?